### ISSUE TITLE
在进行reranker之后 计算reranker.compute_score(['query', 'passage']) 分数居然是一样的

### ISSUE THREAD

**Issue by wang-ship-it** [FlagOpen/FlagEmbedding#864]
CUDA_VISIBLE_DEVICES=4,5,6 nohup torchrun --nproc_per_node 3 -m FlagEmbedding.reranker.run --output_dir saved_model_bgererank --model_name_or_path ./bge-reranker-v2-m3 --train_data ./data/finall_pair_with_label_for_bgererank.jsonl --learning_rate 6e-5 --fp16 --num_train_epochs 20 --per_device_train_batch_size 1 --gradient_accumulation_steps 4 --dataloader_drop_last True --train_group_size 2 --max_len 1024 --weight_decay 0.01 --logging_steps 10 --log_level error --save_steps  1000 > ggg.log 2>&1 &

from FlagEmbedding import FlagReranker

reranker = FlagReranker('/dockerdata/FlagEmbedding-master/saved_model_bgererank/checkpoint-80000')

scores = reranker.compute_score([['what is panda?', 'hi'], ['what is panda?', 'The giant panda (Ailuropoda melanoleuca), sometimes called a panda bear or simply panda, is a bear species endemic to China.']], normalize=True)
print(scores)

[0.3530541967332432, 0.3530541967332432]

它们的分数居然是一样的

**Comment by staoxiao** [comment]
Please check the training log. Do training losses go down normally?

**Comment by wang-ship-it** [comment]
> Please check the training log. Do training losses go down normally?

loss是不正常的 感觉是数据的问题了 感谢

{'loss': 0.6925, 'grad_norm': 2.0849599838256836, 'learning_rate': 1.287668960942035e-06, 'epoch': 19.58}
{'loss': 0.6932, 'grad_norm': 2.070359468460083, 'learning_rate': 1.2851811924703542e-06, 'epoch': 19.58}
{'loss': 0.6923, 'grad_norm': 1.974111795425415, 'learning_rate': 1.2826934239986732e-06, 'epoch': 19.58}
{'loss': 0.6952, 'grad_norm': 2.028146266937256, 'learning_rate': 1.2802056555269925e-06, 'epoch': 19.58}
{'loss': 0.6904, 'grad_norm': 2.020315408706665, 'learning_rate': 1.2777178870553115e-06, 'epoch': 19.58}
{'loss': 0.6908, 'grad_norm': 2.1246161460876465, 'learning_rate': 1.2752301185836303e-06, 'epoch': 19.58}
{'loss': 0.6928, 'grad_norm': 2.1331875324249268, 'learning_rate': 1.2727423501119496e-06, 'epoch': 19.59}
{'loss': 0.695, 'grad_norm': 2.0315945148468018, 'learning_rate': 1.2702545816402686e-06, 'epoch': 19.59}
{'loss': 0.6929, 'grad_norm': 2.073537826538086, 'learning_rate': 1.2677668131685879e-06, 'epoch': 19.59}
{'loss': 0.6928, 'grad_norm': 2.1141178607940674, 'learning_rate': 1.265279044696907e-06, 'epoch': 19.59}
{'loss': 0.6908, 'grad_norm': 2.0629518032073975, 'learning_rate': 1.262791276225226e-06, 'epoch': 19.59}
{'loss': 0.696, 'grad_norm': 2.071342706680298, 'learning_rate': 1.2603035077535452e-06, 'epoch': 19.59}
{'loss': 0.6963, 'grad_norm': 2.1142327785491943, 'learning_rate': 1.257815739281864e-06, 'epoch': 19.59}
{'loss': 0.6907, 'grad_norm': 2.0664267539978027, 'learning_rate': 1.2553279708101833e-06, 'epoch': 19.59}
{'loss': 0.6938, 'grad_norm': 2.072291612625122, 'learning_rate': 1.2528402023385023e-06, 'epoch': 19.59}
{'loss': 0.6936, 'grad_norm': 2.1236541271209717, 'learning_rate': 1.2503524338668216e-06, 'epoch': 19.59}
{'loss': 0.6919, 'grad_norm': 2.235614776611328, 'learning_rate': 1.2478646653951406e-06, 'epoch': 19.59}
{'loss': 0.6902, 'grad_norm': 2.1468000411987305, 'learning_rate': 1.2453768969234597e-06, 'epoch': 19.59}
{'loss': 0.6961, 'grad_norm': 2.118581771850586, 'learning_rate': 1.242889128451779e-06, 'epoch': 19.6}
{'loss': 0.6922, 'grad_norm': 2.0804717540740967, 'learning_rate': 1.2404013599800978e-06, 'epoch': 19.6}
{'loss': 0.6977, 'grad_norm': 2.1656246185302734, 'learning_rate': 1.237913591508417e-06, 'epoch': 19.6}
{'loss': 0.6916, 'grad_norm': 2.129641532897949, 'learning_rate': 1.235425823036736e-06, 'epoch': 19.6}
{'loss': 0.6965, 'grad_norm': 2.0003018379211426, 'learning_rate': 1.2329380545650551e-06, 'epoch': 19.6}
{'loss': 0.6911, 'grad_norm': 2.1318087577819824, 'learning_rate': 1.2304502860933744e-06, 'epoch': 19.6}
{'loss': 0.696, 'grad_norm': 2.0877270698547363, 'learning_rate': 1.2279625176216934e-06, 'epoch': 19.6}
{'loss': 0.6923, 'grad_norm': 1.9669597148895264, 'learning_rate': 1.2254747491500127e-06, 'epoch': 19.6}
{'loss': 0.692, 'grad_norm': 2.0027406215667725, 'learning_rate': 1.2229869806783315e-06, 'epoch': 19.6}
{'loss': 0.6934, 'grad_norm': 2.104949951171875, 'learning_rate': 1.2204992122066505e-06, 'epoch': 19.6}
{'loss': 0.6915, 'grad_norm': 2.07004714012146, 'learning_rate': 1.2180114437349698e-06, 'epoch': 19.6}
{'loss': 0.6928, 'grad_norm': 2.086940288543701, 'learning_rate': 1.2155236752632888e-06, 'epoch': 19.6}
{'loss': 0.6933, 'grad_norm': 2.0461232662200928, 'learning_rate': 1.213035906791608e-06, 'epoch': 19.61}
{'loss': 0.6947, 'grad_norm': 2.073122501373291, 'learning_rate': 1.2105481383199271e-06, 'epoch': 19.61}
{'loss': 0.6944, 'grad_norm': 2.028000831604004, 'learning_rate': 1.2080603698482462e-06, 'epoch': 19.61}
{'loss': 0.6943, 'grad_norm': 2.1740689277648926, 'learning_rate': 1.2055726013765652e-06, 'epoch': 19.61}
{'loss': 0.6957, 'grad_norm': 2.034731149673462, 'learning_rate': 1.2030848329048842e-06, 'epoch': 19.61}
{'loss': 0.6901, 'grad_norm': 2.0066068172454834, 'learning_rate': 1.2005970644332035e-06, 'epoch': 19.61}
{'loss': 0.6966, 'grad_norm': 2.06553053855896, 'learning_rate': 1.1981092959615225e-06, 'epoch': 19.61}
{'loss': 0.6931, 'grad_norm': 2.045304775238037, 'learning_rate': 1.1956215274898418e-06, 'epoch': 19.61}
{'loss': 0.6934, 'grad_norm': 2.0278573036193848, 'learning_rate': 1.1931337590181608e-06, 'epoch': 19.61}
